### ISSUE TITLE
Reward of 1/2 ETH to anyone solving my issue.

### ISSUE THREAD

**Issue by babu2001** [ethereum-mining/ethminer#360]
Hi guys. I prefer to pay than losing more of my time on this. 1/2 ETH to the one solving this issue for real. I have 3 rigs and all of them have the same install. Windows 10 updated and nothing else on it but Claymore. I have a BTC 110+ hooked to 13 EVGA 1060 GTX 3GB. I'm trying to get a little more juice from them and everybody is pointing me MSI AfterBurner. With the latest driver build 388, latest Windows build 1709 and the latest AfterBurner, My issue is when I run the software, it won't load at all. I can see it in the task manager but only for a couple of seconds and then it vanishes. I'm running it as admin. Does someone have a clue on this one? Thanks for any help.

**Comment by babu2001** [comment]
BTW, I've tried also the latest beta of AB. Of course, EVGA Precision OC (who's telling me "Too many GPU"). I've tried Asus GPU Tweak II (loads but crashes) and NZXT CAM (seems to load but stays on the task bar. Can't login nor change settings)

**Comment by aenciso** [comment]
You're better off Windows. Ubuntu is better on this case. I can give you advice on this.

**Comment by Simon0Harms** [comment]
@aenciso 
/signed
Can Windows manage so many graphic cards?

I would use linux too, if i could change the mem clock of my vega56...

**Comment by Bharathvasan** [comment]
Here is you, Windows 10 with 110+ can able to run only max 7GPU of NVIDIA  and 6 AMD card, you can't run 13 NVIDIA or 13 AMD cards on Windows. either you need to move to Linux or ETHOS or Simple mining OS.
These linux based OS can handle single version of card.

**Comment by Simon0Harms** [comment]
@Bharathvasan 
Since the last AMD driver update you can use up to 12 AMD Graphic Cards with Windows 10

**Comment by Bharathvasan** [comment]
@Simon0Harms No me to had same issue when i was issue windows build on 110+, finally as i said have to move to linux builds to make it work

**Comment by babu2001** [comment]
YES, Windows CAN manage 13 Nvidia at the same time. 1709 build does as should 1703. But I wasn't able with 1511. Everything works fine with 13 cards on Windows. Only the programs to overclock refuse to load.

**Comment by ddobreff** [comment]
try nvoc
https://github.com/deathcamp/NVOC
it applies clocks via nvapi and uses gpu-idx.

**Comment by aenciso** [comment]
@babu2001 that's what I'm saying, afterburner and any other overclocking might have problems overclocking, you should try linux, I can help you with that.

**Comment by GreenRiver69** [comment]
Same issue, worked fine with 8 Nvidia and 5 AMD cards but won't even launch with 13 Nvidia, windows is fine with 13 Nvidia with latest drivers (388) and 1709 FCU, but MSI AB Beta 19 and 4.4.0 won't launch.  "Try Linux" isn't an answer, I have several rigs on Linux based systems and want this on Windows IF I can find a way to overclock/UV?

**Comment by babu2001** [comment]
I gave a try to nvoc but got unspecied error failure and ehtminer64 closing itself. With only +50 CPU and +100 RAM. I don't think it works very well with my cards 'cause these cards can handle a lot more. My HTPC is running at +195 CPU and + 405 RAM... Anyway, looks like I'm stock at those speeds... I'll wait until I got a deal for 5 Radeon 470. Will do a setup like greenriver69 "8 + 5". Thanks for all your inputs guys
